### ISSUE TITLE
Where are initial conditions stored for petrinet?

### ISSUE THREAD

**Issue by jClugstor** [DARPA-ASKEM/Model-Representations#73]
In the SIR examples in petrinet/examples, under `initials` there are symbolic expressions for the initial conditions. The actual numbers for the initial conditions are under `parameters`. In the halfar ice model example, and in the models in `simulation-integration` the numbers for the initial conditions are under `initials` and not parameters.

So which ones need to be changed?

**Comment by bgyori** [comment]
I agree that consistency among different representations here would be advantageous. One clarification on the petrinet (and regnet, stockflow) initials: they are expressions so all of '5.0', 'N - (S0 + I0)', 'S0' are valid there. They aren't just references to parameters.

**Comment by jClugstor** [comment]
Oh I see. That clears it up for me. So the initial conditions in general can be an expression of some of the parameters.

**Comment by bgyori** [comment]
Yes, so when you work with them downstream, you can treat them as expressions that you substitute parameters into to end up with a specific number that you can use for simulation.

**Comment by jClugstor** [comment]
Got it, thanks!
